app/furniture/marketplace/locales/en.yml

```diff
@@ -70,6 +70,7 @@ en:
         success: "Notification Method '%{contact_location}' Saved!"
       destroy:
         success: "Notification Method '%{contact_location}' Removed!"
+        link_to: "Remove Notification Method '%{contact_location}'"
```

Comment:
Hitting a different bug when running testing `spec/furniture/marketplace/marketplace_tags_system_spec.rb` surfaced this missing translation. 🤷‍♀️

Reply:
Probably worth submitting independently so it can be merged without being hung up on feedback for this feature.